03_Modules/Credentials/src/module/api.ts

```diff
@@ -172,4 +173,24 @@ export class CredentialsModuleApi
       : clientInformation;
     return CredentialsResponse.build(toCredentialsDTO(clientInformation));
   }
+
+  @Delete('/delete-tenant/:tenantId')
+  @ResponseSchema(OcpiEmptyResponse, {
+    statusCode: HttpStatus.OK,
+    description: 'Successful response',
+    examples: {
+      success: {
+        summary: 'A successful response',
+        value: MOCK_EMPTY,
+      },
+    },
+  })
+  async deleteTenant(
```

Comment:
Is there a separate PR with an endpoint that allows you to deregister a specific MSP rather than delete the whole CPO tenant? I'm not seeing one--deleting a whole CPO tenant is a nice extra feature but what we really needed is a way to make a specific CPO call DELETE for a specific MSP, then remove that MSP's access to that CPO

Reply:
I see, I think I must have mistunderstood. I will leave this endpoint and add an endpoint to unregister one MSP so that a given CPO can use this endpoint to unregister an MSP.

Reply:
PR updated with the new `/unregister-client` endpoint